sphinx/directives/other.py

```diff
@@ -39,6 +39,7 @@ class TocTree(Directive):
     final_argument_whitespace = False
     option_spec = {
         'maxdepth': int,
+        'name': str,
```

Comment:
optional_arguments is better to me. docutils' contents directive is similar purpose and it take a optional argument as 'title' for contents directive.
http://docutils.sourceforge.net/docs/ref/rst/directives.html#table-of-contents